packages/commuter-server/src/routes/index.js

```diff
@@ -6,9 +6,17 @@ const express = require("express"),
   path = require("path"),
   router = express.Router();
 
-router.use("/api", require("./api"));
+const createAPIRouter = require("./api");
 
-router.use("/files", require("./files"));
+// TODO: Accept content provider choice
+// TODO: Load the configured router
+
+const contentsProvider = require("../content-providers/s3");
```

Comment:
The plan after this is to case off which contentsProvider is selected based on the original options. They both obey the same interface -- now it's just a toggle to set them up.